### DISCUSSION TITLE
Automating Instagram Posts PHP

### DISCUSSION THREAD

**Discussion by jlfinvt** [jstolpe/instagram-graph-api-php-sdk#5] [Q&A]
Hi @jstolpe, thanks for the great work on the instagram-graph-api-php-sdk. I am trying to develop an automated process to have daily images posted to instagram via cron job (php).  I was reviewing your examples from your youtube playlist.  It is a multistep process of logging in to the linked Facebook account, getting the user access token with a callback etc.  Do you have any thoughts on how this could be done without the user intervention ?
Thanks!

**Comment by jstolpe** [comment]
Thanks for checking out the SDK! Unfortunately, in order to post on a users behalf the user has to authorize your application and they need to be a business or creator account as well.

**Reply by jlfinvt** [reply]
Thank you for the reply. If I understand what you are saying, in order to obtain access tokens from IG to do the daily automated posting it would require a business/creator account and having the application reviewed and approved by IG.

**Reply by jstolpe** [reply]
yeah that is correct

**Reply by jlfinvt** [reply]
Thanks again. I really appreciate it.
